src/main/java/seedu/trackermon/commons/core/JarTools.java

```diff
@@ -4,6 +4,10 @@
 import java.io.UnsupportedEncodingException;
 import java.net.URL;
 
+/**
+ * A functionality class that interacts different based on the running application
```

Comment:
Change to `Interacts differently depending if the type of running application is a Jar file or an editor.`?

Reply:
Isn't this a class header, the verb rule only applies to methods header?

Reply:
I'm not 100% sure if it violates the coding standard, but we should still make it consistent with the headers from other classes.